Makefile

```diff
@@ -639,7 +638,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             drivers/rx_nrf24l01.c \
             drivers/rx_spi.c \
             drivers/rx_xn297.c \
-            drivers/pwm_output.c \
```

Comment:
The separation would induce exposure of static objects; make `pwm_impl.h` (`pwm_hidden.h?`) to hide them from rest of the world?